our internal patch to keep bios track.

not for external use.

do not apply.

                ---------------------------------

        Adam Sulmicki <adam@cfar.umd.edu>
        Wed Nov  6 04:43:05 EST 2002
        http://www.eax.com/patches/

-------------------------------------------------------------------------------

Index: rombios.c
===================================================================
RCS file: /fs/cvsroot/loader/bochs/bios/rombios.c,v
retrieving revision 1.3
diff -u -r1.3 rombios.c
--- rombios.c	6 Nov 2002 09:01:13 -0000	1.3
+++ rombios.c	6 Nov 2002 09:06:11 -0000
@@ -937,6 +937,9 @@
 
 /* Offset to skip the CVS $Id: prefix */ 
 #define bios_version_string  (CVSID + 4)
+static char bios_info_string[] = "BOCHS BIOS VER:1.XX";
+static char bis1[] = __DATE__;
+static char bis2[] = __TIME__;
 
 #define BIOS_PRINTF_HALT     1
 #define BIOS_PRINTF_SCREEN   2
@@ -951,11 +954,11 @@
 // BX_DEBUG goes to INFO port until we can easily choose debug info on a 
 // per-device basis. Debug info are sent only in debug mode
 #if DEBUG_ROMBIOS
-#  define BX_DEBUG(format, p...)  bios_printf(BIOS_PRINTF_INFO, format, ##p)    
+#  define BX_DEBUG(format, p...)   printf(format, ##p)
 #else
 #  define BX_DEBUG(format, p...) 
 #endif
-#define BX_INFO(format, p...)   bios_printf(BIOS_PRINTF_INFO, format, ##p)
+#define BX_INFO(format, p...)   printf(format, ##p)
 #define BX_PANIC(format, p...)  bios_printf(BIOS_PRINTF_DEBHALT, format, ##p)
 
 #if DEBUG_ATA
@@ -1747,6 +1750,8 @@
 
   printf(BX_APPNAME" BIOS, %d cpu%s, ", BX_SMP_PROCESSORS, BX_SMP_PROCESSORS>1?"s":"");
   printf("%s %s\n", bios_cvs_version_string, bios_date_string);
+  printf("[%s]\n", bios_info_string);
+  printf("[COMPILE DATE:%s TIME:%s]\n", bis1, bis2);
   printf("\n");
 }
 
